csharp/src/Ice/ConnectionFactory.cs

```diff
@@ -668,7 +633,7 @@ private async ValueTask AcceptAsync()
                         }
                     }
                     _communicator.Logger.Error($"failed to accept connection:\n{ex}\n{_acceptor}");
-                    await Task.Delay(1000).ConfigureAwait(false); // Retry in 1 second
+                    await Task.Delay(1000); // Retry in 1 second
```

Comment:
use a TimeSpan, and remove the comment?
```suggestion
                    await Task.Delay(TimeSpan.FromSeconds(1));
```